src/util.js

```diff
@@ -83,6 +83,7 @@ export function parseCoord(coord) {
 	if (coord.indexOf(".") > 0) {
 		// Reduce a coordinate of a certain color space until the color is in gamut
 		let [spaceId, coordName] = coord.split(".");
+		// eslint-disable-next-line no-undef
 		let space = Color.space(spaceId);
```

Comment:
Where is the undeclared variable here?

Reply:
`Color`, which is not quite a false positive...

Reply:
Whoa, it looks like this file does not import `Color` anywhere! Does it depend on having `Color` as a global variable? If so this will break in most bundles! We should be fixing this instead of adding comments to get ESLint to ignore it…

Reply:
It... does! The tricky thing is that `util` is imported by `color`, so I guess it needs to go into `color.js`??

Reply:
Yikes. I can't remember what happens if two ES modules import each other, does it create a loop or does the engine do something clever?